#162 fix `k3d.collectAndArchiveLogs` to not fail when a logfile cannot be deleted (#163)
cloudogu/ces-build-lib: CHANGELOG.md

## [Unreleased]
### Fixed

> - [#162] Fix `k3d.collectAndArchiveLogs` to not fail when a logfile cannot be deleted.

Reviewer: ```suggestion
- [#162] Fix `k3d.collectAndArchiveLogs` to not fail when a logfile to archive was not created
```
